src/mysql/connection.cr

```diff
@@ -6,7 +6,7 @@ class MySql::Connection < DB::Connection
     @socket = uninitialized TCPSocket
 
     begin
-      host = context.uri.host.not_nil!
+      raise "no host provided" unless host = context.uri.hostname
```

Comment:
`Exception` message should always be _Sentence cased_.

Reply:
I followed the already existing exception below ` raise "packet #{status} not implemented"`